pkg/api/v1beta2/types.go

```diff
@@ -1094,3 +1102,23 @@ type ResourceQuotaList struct {
 	// Items is a list of ResourceQuota objects
 	Items []ResourceQuota `json:"items"`
 }
+
+// Secret holds secret data of a certain type
+type Secret struct {
+	TypeMeta `json:",inline"`
+
+	Data map[string][]byte `json:"data,omitempty"`
+	Type SecretType        `json:"type,omitempty"`
+}
+
+type SecretType string
+
+const (
+	SecretTypeOpaque SecretType = "opaque" // Default; arbitrary user-defined data
```

Comment:
Please capitalize. All constants in the API are CamelCase.

Please tell me if api-conventions.md is not clear.